playground/AspireEventHub/EventHubsConsumer/EventHubsConsumer.csproj

```
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

```suggestion
<Project Sdk="Microsoft.NET.Sdk.Worker">
```

Then you can also delete the `<OutputType>exe</OutputType>` line below as well.